deployment/network-operator/templates/operator.yaml

```diff
@@ -46,9 +56,38 @@ spec:
       serviceAccountName: {{ include "network-operator.fullname" . }}
       imagePullSecrets: {{ include "network-operator.operator.imagePullSecrets" . }}
       containers:
+        {{- if .Values.admissionController.enabled }}
+        - name: kube-rbac-proxy
+          args:
+          - --secure-listen-address=0.0.0.0:8443
+          - --upstream=http://127.0.0.1:8080/
```

Comment:
ah this guy is for securing metrics endpoint

which is 8080 in our operator.

Reply:
Removed